src/config/index.ts

```diff
@@ -58,6 +58,9 @@ export const getTxServiceUriFrom = (safeAddress) =>
 export const getIncomingTxServiceUriTo = (safeAddress) =>
   `safes/${safeAddress}/incoming-transfers/`
 
+export const getAllTxServiceUriTo = (safeAddress) =>
```

Comment:
can we call it `getTransactionsNewServiceUriTo`?. Sorry to bother with this, but later, when we need to replace the old code, I would like to search for `transactionsNew` and have an easy way to see all the places where we need to rename it.

Reply:
yep that makes sense! will do it

Reply:
nitpick: `getTransactionsNewServiceUriTo` doesn’t sound nice, let’s call it `getAllTransactionsServiceUriFrom` or `getNewTransactionsServiceUriFrom`. In the first case you don’t even have to change anything once the new page is implemented

Reply:
You can even go further and drop “Service” from the name

Reply:
> In the first case you don’t even have to change anything once the new page is implemented
I think the `All` is irrelevant, by naming it `transactions` you can assume that it will return all transactions.

Reply:
I renamed it to `getNewTransactionsServiceUriFrom` :D